dashboard/test/controllers/api/v1/user_school_infos_controller_test.rb

```diff
@@ -0,0 +1,99 @@
+require 'test_helper'
+
+class UserSchoolInfosControllerTest < ActionDispatch::IntegrationTest
+  test "last confirmation date in user school infos table is updated" do
+    user_school_info = create :user_school_info
+    sign_in user_school_info.user
+    patch "/api/v1/user_school_infos/#{user_school_info.id}/update_last_confirmation_date"
+    assert_response :success
+
+    assert user_school_info.last_confirmation_date.to_datetime, DateTime.current
+  end
+
+  test "will redirect user to sign in" do
+    patch "/api/v1/user_school_infos/-1/update_last_confirmation_date"
+    assert_response 302
+  end
+
+  test "last confirmation date will 404 if user id does not exist" do
+    user_school_info = create :user_school_info
+    sign_in user_school_info.user
+    patch "/api/v1/user_school_infos/-1/update_last_confirmation_date"
+    assert_response 404
+  end
+
+  test 'update last confirmation date will 401 if given a user id other than the person logged in' do
```

Comment:
```suggestion
  test 'update last confirmation date will 401 if given a school_info id not owned by the signed-in user' do
```

Reply:
there's a couple other tests below that are also falsely asserting that user id is being used

Reply:
Thanks!